docs/datasuite.md

```diff
@@ -0,0 +1,152 @@
+# Data Suite
+
+ConnexCS's Data Management Suite helps you manage, organize, and optimize your data throughout its lifecycle.
+
+Our suite  includes software solutions for data collection, storage, manipulation, editing, modify, processing, analysis, governance, and security.
+
+It will enable your businesses to handle large amounts of structured and unstructured data efficiently, ensuring that data is accessible, accurate, and secure.
+
+What our Data Suite offers?
+
+1. Data Integration
+2. Data Storage
+3. Data Governance
+4. Data Security and Privacy
+5. Data Analytics and Reporting
+6. Data Archiving
+
+## Steps to use Data Management Suite
+
```

Comment:
Create separate sections 
Like section i how to create 
2 How to edit 
3 how to upload 
3 server side queries 
So that it is clear what we are doing

Reply:
doubt pending with Adam